spec/support/negated_matcher.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+RSpec::Matchers.define_negated_matcher :not_change, :change
```

Comment:
What's that for? 😃

Reply:
This is because you can’t use `not_to` with compound expectations :sweat_smile: 
See https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecchangebyzero and https://relishapp.com/rspec/rspec-expectations/v/3-11/docs/define-negated-matcher :slightly_smiling_face:

Reply:
K, got it. Is it possible to add that to [rubocop-discourse](https://github.com/discourse/rubocop-discourse)?

Reply:
I think it fits well. Imo the idea is: add this gem, and you can write tests the same as you do in core.

If we use them here, I don't see any reason to not use them in e.g. discourse-calendar 😃 (without copy and paste, that is 😅)